README.md

```diff
@@ -93,7 +93,7 @@ Flags:
                                   socket. Leave this empty to use the defaults.
       --profiling-duration=10s    The agent profiling duration to use. Leave
                                   this empty to use the defaults.
-      --systemd-cgroup-path="/sys/fs/cgroup/systemd/system.slice"
+      --systemd-cgroup-path=STRING
```

Comment:
I think this is a left-over change from the other PR #206. Let's keep things clean.